spec/std/system_spec.cr

```diff
@@ -18,4 +18,12 @@ describe System do
       cpu_count.should eq(shell_cpus)
     end
   end
+
+  describe "login" do
+    it "returns String when found" do
+      shell_login = `echo $USER`.strip
```

Comment:
Oh, just realised. `` `logname`.strip `` is the command equivalent to `getlogin`, not `ENV["USER"]?`. Sorry for making you jump through hoops! You'll also need to convert "logname: no login name" to `nil` to pass the test on travlis.